ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/questionnaire-completed/questionnaire-completed.component.html

```diff
@@ -3,12 +3,116 @@
     <h1 class="govuk-heading-xl" i18n="@@questionnaireCompleted_h1">
       Questionnaire completed
     </h1>
+    <div>
+      <app-hearing-details-header></app-hearing-details-header>
+    </div>
+    <div>
+      <p class="govuk-body">Here's a summary of the information you've provided.</p>
+    </div>
+
+    <h2 class="govuk-heading-m">About you</h2>
+    <table class="govuk-table" style="width:100%;">
+      <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <td [ngClass]="{'govuk-table__cell': !model.aboutYou.answer}">
```

Comment:
this looks like a recurring pattern, maybe it can be wrapped with something? like even:
`<td [class]="highlightIfSet(!model.aboutYou.answer)">`

That would add some context to why you are setting/not setting the `govuk-table__cell` style as well

Reply:
Changed to css tr:last-child

Reply:
cool, that's a lot cleaner!